File: 1-two-sum/two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        d={}
        #d=val:index
        #diff=0
        for i,v in enumerate(nums):
            #dif=target-v
            if target- v in d:
                return [d[target-v],i]
            else:
                d[v]=i
        

File: 1-two-sum/two-sum.dart
class Solution {
  List<int> twoSum(List<int> nums, int target) {
    List<int>? ans = [];
    for (var i = 0; i < nums.length-1; i++){
        for (var j =i+1; j< nums.length; j++){
            if (nums[i]+nums[j]== target){
                ans.add(i);
                ans.add(j);
            }
            }
    }
    return ans;

  }
}